mothulity/mothulity

```diff
@@ -334,6 +334,22 @@ def main():
         Default <~/db/Silva.nr_v119/silva.nr_v119.tax>.
         """
     )
+    mothur.add_argument(
+        "--alignment-region",
+        action="store",
+        dest="alignment_region",
+        metavar="6388-13861",
+        default=False,
+        help="""
+        Cuts choosen alignment database to desired region.
+        Available options: v3, v4, v3v4, <start-end>.
+        In case of passing start-end posittion instead of region name,
+        coordinates must be separate by '-'.
+        In order to determine the coordinates perfectly matching used primers
+        it is strongly recommended to follow PD Schloss guide
+        (available on mothur blog).
```

Comment:
Please use the actual link to the blog if possible.

Reply:
Link added.